tests/graphql/transaction.spec.ts

```diff
@@ -173,7 +173,7 @@ describe("Transaction", () => {
     });
   });
 
-  describe("#categorize", () => {
+  describe("#update", () => {
```

Comment:
shouldn't we leave old tests? because method will still be used for a while

Reply:
Oups right. I'll put them back.